source/runner/Executor.ts

```diff
@@ -399,22 +399,28 @@ export class Executor {
         }
       }
       if (promise) {
-        commentPromises.push(promise.then(_r => emptyDangerResults).catch(_e => inlineResultsIntoResults(inlineResult)))
+        commentPromises.push(
+          promise.then((_r) => emptyDangerResults).catch((_e) => inlineResultsIntoResults(inlineResult))
+        )
       }
     }
-    deleteComments.forEach(comment => {
+    deleteComments.forEach((comment) => {
       let promise = this.deleteInlineComment(comment)
-      commentPromises.push(promise.then(_r => emptyDangerResults).catch(_e => emptyDangerResults))
+      commentPromises.push(promise.then((_r) => emptyDangerResults).catch((_e) => emptyDangerResults))
     })
 
     return Promise.all([
-      this.sendInlineReview(git, inlineResultsForReview).catch(_e =>
-        inlineResultsForReview.forEach(inlineResult => inlineResultsIntoResults(inlineResult))
-      ),
+      this.sendInlineReview(git, inlineResultsForReview)
+        .then((_r) => emptyDangerResults)
+        .catch((_e) =>
+          inlineResultsForReview
+            .map((inlineResult) => inlineResultsIntoResults(inlineResult))
+            .reduce(mergeResults, emptyResult)
+        ),
```

Comment:
Great fix! I had a suspicion the root issue was around here, but the 3 times I traced it, I couldn’t figure it out — indeed it looked like everything was being caught & handled..